packages/compute-baseline/README.md

```diff
@@ -92,17 +117,12 @@ If you don't want to import `@mdn/browser-compat-data` as your data source, you
 
 ```javascript
 import data from "some-parsed-json-file";
-import { computeBaseline } from "compute-baseline";
+import { getStatus } from "compute-baseline";
 import { Compat } from "compute-baseline/browser-compat-data";
 
 const compat = new Compat(data);
 
-computeBaseline(
-  {
-    compatKeys: ["css.properties.border-color"],
-  },
-  compat,
-);
+getStatus("fetch", "api.Response.json", compat);
```

Comment:
Can you help me understand when someone would need to use their own compat data? Are there other data sources out there that match the BCD schema and which `getStatus` would be able to work with?

If this is a valid use case, then I would suggest not using the `getStatus` method for it, since `getStatus` is for blessed/editorial reviewed Baseline statuses only. But if you use your own data, then we can't guaranty anything about it.

Reply:
Yes, I was trying to accommodate (for example) MDN upgrading BCD independently of web-features. Really, I'd like to be able to say something like: "You can use any version of BCD >= web-features's version of BCD" but we don't currently provide that information to consumers of web-features.

Another route here is to leave this usage undocumented.